### PR TITLE
shows globe icon for online workshops

### PR DESCRIPTION
**Pull request by maneesha**
This is the proposed way that online workshops will show on our website.  It adds a globe icon next to the country flag for online workshops.  If no country is listed for an online workshop, only the globe icon appears.  

@sheraaronhurt  what do you think on behalf of the workshop admin team? Would you also want to see the text "(online)" appear next to the venue name? 

image

### CODE REVIEW

File: _includes/workshop_table.md [hunk 1]
```diff
@@ -25,10 +25,16 @@
...
     </td>
+
     <td>
-        {% unless w.country == "" %}
+        {% unless w.country == "" or w.country == "W3" %}
       <img src="{{site.url}}/assets/img/flags/{{site.flag_size}}/{{w.country | downcase}}.png" title="{{w.country | replace: '-', ' '}}" alt="{{w.country | replace: '-', ' ' | downcase}}"  class="flags"/>
       {% endunless %}
+
+      {% if tags contains "online" %}
+      <img src="{{site.url}}/assets/img/flags/{{site.flag_size}}/w3.png" title="Online" alt="{{w.country | replace: '-', ' ' | downcase}}"  class="flags"/>
```
Comment (fmichonneau) [inline _includes/workshop_table.md]:
```suggestion
      <img src="{{site.url}}/assets/img/flags/{{site.flag_size}}/w3.png" title="Online" alt="globe image" class="flags"/>
```

**Review by fmichonneau — APPROVED** [review]
this looks good to me!

**Comment by fmichonneau** [comment]
> Would you also want to see the text "(online)" appear next to the venue name?

if instructors use the latest version of the template and follow our recommendations, "(online)" will appear in the name of the venue in this list.

**Comment by sheraaronhurt** [comment]
@maneesha @fmichonneau this looks great! I had no preference about (online) but since it's there. I think that's great!

**Comment by kariljordan** [comment]
The globe icon looks so great. Wow! I have a question that I can turn into an issue if needed. If a workshop is online, but it's for a specific community (for example, they are running an online workshop but the audience is South Africa). are we doing anything with that?

**Comment by maneesha** [comment]
Great question @kariljordan 

Having both the globe icon and the national flag was intended for that purpose, so we can show that a workshop is taking place online and is hosted by an organization in country X.  

If registration is limited to a specific audience (like people from a specific country or at a particular institution)  we've never actually displayed that on the workshop list on our website. That information is available on the individual workshop webpage.

If you have other ideas on how to display our workshop list, a new issue would be great.

**Comment by kariljordan** [comment]
That is perfect! Thank you @maneesha.

**Pull request merged**